identity-wallet/src/state/reducers/dev_mode.rs

```diff
@@ -38,20 +42,217 @@ lazy_static! {
     );
 }
 
-pub async fn set_dev_mode(state: AppState, action: Action) -> Result<AppState, AppError> {
-    if let Some(enabled) = listen::<SetDevMode>(action).map(|payload| payload.enabled) {
-        return Ok(AppState {
-            dev_mode_enabled: enabled,
-            current_user_prompt: None,
-            ..state
-        });
+const PROFILE_PW: &str = "sup3rSecr3t";
```

Comment:
I would generally try to avoid potentially ambiguous abbreviations throughout the code and rather spell it out: `PROFILE_PASSWORD` or even just `PASSWORD`.

Reply:
fixed